### PR TITLE
Setup vetclinic

### PR DESCRIPTION
**Pull request by Newtayo**
**In this Pull Request, I implement the following: **
- [ ] Created a database named vet_clinic.
- [ ] Created a table named animals inside the database vet_clinic.
- [ ] Query the table using several criteria. Kindly click on the criteria to see the picture of the query results
         [ -     Find all animals whose name ends in "mon" ](https://github.com/Newtayo/Vet-clinic/blob/setup-vetclinic/SQL_QUERY/animals_mon_names.JPG)  
          [ -     List the name of all animals born between 2016 and 2019.](https://github.com/Newtayo/Vet-clinic/blob/setup-vetclinic/SQL_QUERY/Animals_between_2016_to_2019.JPG)
           [-     List the name of all animals that are neutered and have less than 3 escape attempts.](https://github.com/Newtayo/Vet-clinic/blob/setup-vetclinic/SQL_QUERY/neutered_and_attempted_escape.JPG)
           [-     List the date of birth of all animals named either "Agumon" or "Pikachu".](https://github.com/Newtayo/Vet-clinic/blob/setup-vetclinic/SQL_QUERY/DOB_pikachu_Agumon.JPG)
           [-     List name and escape attempts of animals that weigh more than 10.5kg](https://github.com/Newtayo/Vet-clinic/blob/setup-vetclinic/SQL_QUERY/weight_is_above_10.5.JPG)
         [  -     Find all animals that are neutered.](https://github.com/Newtayo/Vet-clinic/blob/setup-vetclinic/SQL_QUERY/neutered%20animals.JPG)
           [-     Find all animals not named Gabumon.](https://github.com/Newtayo/Vet-clinic/blob/setup-vetclinic/SQL_QUERY/not%20Gabumon.JPG)
           [-     Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals 
                 precisely 10.4kg or 17.3kg)](https://github.com/Newtayo/Vet-clinic/blob/setup-vetclinic/SQL_QUERY/weightbetween17.3and10.4.JPG)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,17 +63,17 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [VET-CLINIC] <a name="about-project"></a>
 
 > Describe your project in 1 or 2 sentences.
```
Comment (rotimiazeez) [inline README.md]: Kindly remove all section description for all your README sections. 👍🏼

**Review by rotimiazeez — COMMENTED** [review]
# Hi @Newtayo, 

### While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Please ensure all your README sections are filled out, you are missing the `future features ` and `show your support section ` and also ensure all section description are remove as they are just to let you know what to do in that section 😉 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

**Comment by Newtayo** [comment]
**In this PR, I removed all description from the readme file**

**Review by binhussen — APPROVED** [review]
Hi @Newtayo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- Descriptive pull request ✔️
- Used Git flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**